src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -56,12 +57,80 @@ public sealed partial class PublisherSettings : ServiceSettingsBase
         /// <returns>A new instance of the default PublisherSettings.</returns>
         public static PublisherSettings GetDefault() => new PublisherSettings();
 
+        public static Predicate<RpcException> IdempotentRetryFilter { get; } =
+            RetrySettings.FilterForStatusCodes(StatusCode.Unavailable, StatusCode.DeadlineExceeded);
+
+        public static Predicate<RpcException> NonIdempotentRetryFilter { get; } =
+            RetrySettings.FilterForStatusCodes();
```

Comment:
This doesn't seem the negation of the `IdempotentRetryFilter`, if I read this correctly is no status code?